content/applications/hr/recruitment.rst

```diff
@@ -26,52 +26,172 @@ is able to be modified to suit the specific recruitment flow for any business.
+app --> Settings`. After any changes are made, click the :guilabel:`Save` button in the top-left
+corner to save all the changes.
+
+Job posting
+-----------
+
+The *Job Posting* section of the *Recruitment* app settings has only one selection to make. If job
+positions are to be posted to the company's website, enable the :guilabel:`Online Posting` option.
+
+This option only works if the *Website* application is also installed.
+
+Recruitment process
+-------------------
+
+This section of the settings page specifies what the database can and cannot do during the
+recruitment process.
+
+Send interview survey
+~~~~~~~~~~~~~~~~~~~~~
+
+Odoo is capable of having a survey, or exam, sent to an applicant to better determine their skill
+set. This aids the recruitment team and helps them determine which candidates should be moved
+through the recruitment pipeline, and which should no longer be considered.
+
+Enable the :guilabel:`Send Interview Survey` option to be able to send surveys to applicants.
+
+.. note::
+   The *Surveys* application must be installed to use this feature.
+
+Send SMS
+~~~~~~~~
+
+It is possible to send text messages to applicants directly through the *Payroll* app. To do so,
+enable the :guilabel:`Send SMS` option. This option requrtes credits to use. For more information,
+refer to the :doc:`SMS pricing and FAQs
+</applications/marketing/sms_marketing/pricing/pricing_and_faq>` documentation.
+
+.. _recruitment/cv-display:
+
+CV display
+~~~~~~~~~~
+
+When applicants submit an application, one of the default required fields is a resumé, or CV. All
+resumés are stored in the *Documents* application, and are accessible on the applicant's card.
+
+A resumé has the option to appear on the applicant's card, instead of having a document link appear
+in the chatter, where it then needs to be expanded or downloaded to view it.
+
+Enable the :guilabel:`CV Display` option to show the resumé on the applicant's card by default, and
+in addition to the document link. When enabled, the resumé appears on the right side of the
+applicant's card.
+
+If the window or tab is not wide enough, the resumé does not appear. The screen must be wide enough
+to accommodate the resumé. The full-screen view typically accommodates this view.
+
+.. image:: recruitment/cv-display.png
+   :align: center
+   :alt: The resumé on an applicant's card, appearing on the right side.
+
+.. _recruitment/cv-ocr:
+
+CV digitization (OCR)
+~~~~~~~~~~~~~~~~~~~~~
+
+When an application is submitted, it is possible to have Odoo extract the applicant's name, phone
+number, and email address directly from the resumé. To do so, enable the :guilabel:`CV Digitization
+(OCR)` option.
+
+When enabled, additional options appear. Click on the corresponding radio button to select one of
+the following options:
+
+- :guilabel:`Do not digitize`: this option turns off resumé digitization.
+- :guilabel:`Digitize on demand only`: this option only digitizes resumes when requested. A
```

Comment:
Curious: is it possible to digitize in bulk, maybe in a list view group selection?

Reply:
I don't believe so- I tried myself by clicking and draggin, or importing, or creating new and adding several- looks lik eyu can only do one at a time. And online form the application form, only one. The ONYL thing I think could happen is if you emailed the email alias several applications at once, but I don't think that works- I am checking with the experts if there is a way to do that. Although between you, me, and the wall, since it imports such litte informatio (the bare mnimum) it almostdoesn't seem worth it, if the oly way is to email. But I'll update when I hear back! In the meantime, I expanded this a little.